src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -68,6 +68,68 @@ void appendStyled(FlutterLogEntry entry, String text) {
       abstract void render(FlutterLogEntry entry);
     }
 
+    abstract class Column extends ColumnInfo<DefaultMutableTreeNode, FlutterLogEntry> {
+      boolean show = true;
+      private TableCellRenderer renderer;
+
+      Column(String name) {
+        super(name);
+      }
+
+      boolean isVisible() {
+        return show;
```

Comment:
`show` vs `isVisible`
-> I think it should be same.
I mean, both should be `boolean show` && `boolean isShow()`
OR
`boolean isVisible` && `boolean isVisible()`

Reply:
Agreed!